open_humans/storage.py

```diff
@@ -4,7 +4,6 @@
 
 TEN_MINUTES = 60 * 10
 
-
```

Comment:
from PEP8: "Surround top-level function and class definitions with two blank lines."